DataScience/DataScience.py

```diff
@@ -48,25 +45,21 @@ def read(self, eventid):
 
         if not self.file:
             self.file = open(self.joined_data.filename, 'r')
-
-        # continue to read one more event to close the file
+            
+        # read all events in file
         ret = None
         for line in self.file:
             js = json.loads(line)
             js_event_id = js['_eventid']
             if (js_event_id == eventid):
-                ret = [line, js]
+                ret = line
```

Comment:
It's not being used anywhere and it led to a bug (the pop in the first line of the function was returning just the line.